library/src/main/java/com/almeros/android/multitouch/RotateGestureDetector.java

```diff
@@ -155,6 +156,25 @@ protected void resetState() {
         mSloppyGesture = false;
     }
 
+    private float mFocusX;
```

Comment:
Please group these properties with the class properties above.

Reply:
done

Reply:
Additionaly I updated gradle, build tool and compile sdk version.